code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -475,6 +475,10 @@
 
 	if(isobj(A))
 		O = A
+		if(CHECK_BITFIELD(O.resistance_flags, UNACIDABLE|INDESTRUCTIBLE))
```

Comment:
This is not only unatomized if it's supposed to be a fix, but this is already handled through `can_use_ability`